ovos_bus_client/session.py

```diff
@@ -278,11 +288,23 @@ def __init__(self, session_id: str = None, expiration_seconds: int = None,
         @param lang: language associated with this Session
         @param context: IntentContextManager for this Session
         """
+        if tts_prefs:
+            log_deprecation("'tts_prefs' kwarg has been deprecated! value will be ignored", "0.1.0")
+        if stt_prefs:
+            log_deprecation("'stt_prefs' kwarg has been deprecated! value will be ignored", "0.1.0")
```

Comment:
Where should these go instead? Or are they just removed?

Reply:
just removed, this will be handled later when the concept of "user" is finished. dont want to move a whole mycroft.conf around in session. at some point a "user_id" will be added to retrieve this stuff when needed

Reply:
side note, this also leaks api keys in bus messages, thats a big motivation to drop it